include/SFML/Window/Keyboard.hpp

```diff
@@ -153,6 +158,158 @@ class SFML_WINDOW_API Keyboard
         KeyCount      ///< Keep last -- the total number of keyboard keys
     };
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Scancodes
+    ///
+    /// The enumerators are bound to a physical key and do *not* depend
+    /// on the keyboard layout used by the operating system. Usually, the AT-101
+    /// keyboard can be used as reference for the physical position of the keys.
+    ///
+    /// The scancodes are based on a subset of Table 12: Keyboard/Keypad Page
+    /// of Universal Serial Bus (USB): HID Usage Tables, v1.12.
+    ///
+    /// \todo When porting this for SFML 3, remove the `s` prefix and use
```

Comment:
s/s/Scan

Reply:
Does this make sense on the doxygen doc? Maybe just a in code comment? I'm fine either way.

Reply:
Yes, I think it makes sense because it generates a TODO page, centralising all TODOs in one place automatically. And it's easy to grep so it's a win-win situation. :)